### ISSUE TITLE
The command prompt cannot be opened after updating to 1.82.0.0

### ISSUE THREAD

**Issue by xiahuaijia** [ForkIssues/TrackerWin#1815]
"Make Fork a 64-bit application" is mentioned in the changelog, but the path of the command prompt in the software has not been modified.
1.82 cannot open the command prompt. After I found some information and modified it, I can open the command prompt in 1.82

[The 'Sysnative' folder in 64-bit Windows explained](https://www.samlogic.net/articles/sysnative-folder-64-bit-windows.htm)

Fork 1.81
```json
"ShellTool": {
    "Type": "CommandPrompt",
    "ApplicationPath": "C:\\Windows\\sysnative\\cmd.exe"
  },
```

Fork 1.82 expected
```json
"ShellTool": {
    "Type": "CommandPrompt",
    "ApplicationPath": "C:\\Windows\\System32\\cmd.exe"
  },
```

**Comment by DanPristupov** [comment]
Thank you for the bug report

**Comment by DanPristupov** [comment]
I released the hotfix Fork 1.82.1. Please check for updates

**Comment by xiahuaijia** [comment]
Opening in the console (command prompt) is now working
